tests/Avalonia.Controls.UnitTests/TextBoxTests.cs

```diff
@@ -460,6 +460,24 @@ public void Setting_Bound_Text_To_Null_Works()
             }
         }
 
+        [Fact]
+        public void Setting_Bound_Text_To_Null_Works()
```

Comment:
@benthecarman seems like there's a unit test with the same name too. Please add the appropriate unit test name

Reply:
Woops, fixed